### PR TITLE
Don't accept changes not made by humans

### PR DESCRIPTION
**Pull request by mlocati**

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -39,23 +39,15 @@ jobs:
         run: composer update --optimize-autoloader --no-progress --ansi --no-interaction
       - name: Build
         run: ./bin/unipoints build
+      - name: Check built assets
+        if: startsWith(matrix.os, 'windows') == false
+        run: |
+          if ! git diff --exit-code --name-status; then
+            printf '\n\nPLEASE RUN\n\nbin/unipoints build\n'
```
Comment (szepeviktor) [inline .github/workflows/tests.yml]:
```bash
echo "::error file=$(git diff --name-status | head -n 1)::PLEASE RUN bin/unipoints build"
```
that creates a check annotation which is fancy 💄

Comment (szepeviktor) [reply .github/workflows/tests.yml]: image

Comment (mlocati) [reply .github/workflows/tests.yml]: I've added https://github.com/mlocati/unipoints/pull/25/commits/8b94d64fcbf814b05934a8182c197b19237ecb0c to this PR

**Comment by mlocati** [comment]
https://github.com/mlocati/unipoints/commit/8b94d64fcbf814b05934a8182c197b19237ecb0c doesn't seems quite right: https://github.com/mlocati/unipoints/actions/runs/6684901362/job/18162631914#step:6:9

**Comment by szepeviktor** [comment]
You still need your original code.
Put this line before `exit`

**Pull request merged**